llvm/lib/CodeGen/RegAllocFast.cpp

```diff
@@ -584,7 +584,7 @@ void RegAllocFastImpl::spill(MachineBasicBlock::iterator Before,
       SpilledOperandsMap;
   for (MachineOperand *MO : LRIDbgOperands)
     SpilledOperandsMap[MO->getParent()].push_back(MO);
-  for (auto MISpilledOperands : SpilledOperandsMap) {
+  for (auto& MISpilledOperands : SpilledOperandsMap) {
```

Comment:
LLVM style should be `auto &MISpilledOperands`

Reply:
Done, fixed formatting in both files